download/change-log.md

```diff
@@ -4,4 +4,9 @@ title: Change Log
 permalink: /docs/download/change-log
 ---
 
-TODO
+## Change Log
+
+All notable changes to this project will be documented in this file.
+
+### [0.1.0] - 2020-03-01
```

Comment:
It might be better for this to be 04-01 or whenever we make things properly public so it doesn't look like we've been idle for an entire month